tests/ttnn/unit_tests/gtests/ttnn_test_fixtures.hpp

```diff
@@ -67,11 +67,9 @@ class T3kMultiDeviceFixture : public ::testing::Test {
         if (num_devices < 8 or arch != tt::ARCH::WORMHOLE_B0) {
             GTEST_SKIP() << "Skipping T3K Multi-Device test suite on non T3K machine.";
         }
-        const auto T3K_DEVICE_IDS = DeviceIds{0, 4, 5, 1, 2, 6, 7, 3};
```

Comment:
what's the current ordering it now spits out? Is it slightly different ring?

Reply:
I'm not sure actually, but CI was happy 🥇